client/my-sites/plans/navigation.jsx

```diff
@@ -163,4 +169,14 @@ class PlansNavigation extends React.Component {
 	}
 }
 
-export default localize( PlansNavigation );
+export default connect( state => {
+	const siteId = getSelectedSiteId( state );
+	const site = getSite( state, siteId );
```

Comment:
How about using `getSelectedSite` instead, as it already guards for when `siteId` is `null`, without delegating to `getRawSite`/`getSiteBySlug`?

Reply:
That makes total sense when you look at the selector implementations, but it seems like an optimization that `getSite` should implement rather than requiring consumers to know about selector internals.

I'm inclined to leave this as I conceptually prefer the siteId determining the site we select specifically.